extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1260,7 +1266,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
     this.signals()
       .whenSignal(CommonSignals.LOAD_END)
-      .then(() => this.maybeApplyFirstAnimationFrame())
+      .then(() => this.maybeApplyLastAnimationFrame())
```

Comment:
1. Why aren't we applying the first frame anymore?  It's absolutely required for presets.

2. nit: you don't need the `Promise.resolve` below because you're returning from a promise handler already.

```suggestion
      .then(() => this.animationManager_?.applyLastFrame())
```

Reply:
Update: the reason why we need the last animation frame here is because on vertical rendering (which is when this gets called) the first animation frame is not the final state of the page, so the rendering would also be wrong

Reply:
You need to both. The last frame for this wrokaround, and the first frame for presets they'll jump unnecessarily. Something like `applyFirstAndLastFrames`

Reply:
This gets called for the vertical UI (only on bot rendering), so what matters is that we render the final state (aka: the last frame). Actually I found out that we can skip calling `this.animationManager_.finishAll()` if we call `this.animationManager_.applyLastFrame()`